parser_library/src/lsp/lsp_context.cpp

```diff
@@ -648,6 +648,71 @@ bool lsp_context::should_complete_instr(const text_data_view& text, position pos
     return !line_before_continued && std::regex_match(line_so_far.begin(), line_so_far.end(), instruction_regex);
 }
 
+std::vector<std::pair<const context::section*, context::id_index>> gather_reachable_section(
```

Comment:
Only a name change to reflect that it doesn't returns just one section:

```suggestion
std::vector<std::pair<const context::section*, context::id_index>> gather_reachable_sections(
```